packages/paste-website/src/pages/layout/grid/index.mdx

```diff
@@ -592,6 +625,46 @@ Since the grid is sized to the full width of its parent container, you may not w
   </Dont>
 </DoDont>
 
+<DoDont>
+  <Do title="Do" body="Use a vertical grid to create a a stack columns.">
+    <Box padding="space60">
+      <Grid gutter="space30" vertical>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use column span to stack columns.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column span={12}>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column span={12}>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column span={12}>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
```

Comment:
Let's just remove this do don't. 

If you wanna  set column spans to 12 by all means do so. you might have a good reason.